addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -13,6 +13,28 @@ import { unique } from "@web/core/utils/arrays";
 const qweb = core.qweb;
 const _t = core._t;
 
+const allFormsInfo = new Map();
```

Comment:
I don't think cleaning a variable is overkill, it allows to prevent unexpected issues and to always start in the same state (to be consistent) :wink:  Even if I agree that it doesn't cause any issue here

Reply:
Ok I added the clean of the variable :+1: